velox/substrait/SubstraitToVeloxPlan.h

```diff
@@ -22,11 +22,46 @@
 
 namespace facebook::velox::substrait {
 
+class ConnectorHandler {
+ public:
+  virtual ~ConnectorHandler() = default;
+
+  virtual std::shared_ptr<connector::ConnectorTableHandle> createTableHandle(
+      connector::hive::SubfieldFilters filters = {}) = 0;
```

Comment:
SubfieldFilters are HiveConnector-specific.

Reply:
You're correct, I need to generalize this.